libsolidity/interface/AssemblyStack.h

```diff
@@ -21,8 +21,8 @@
 
 #pragma once
 
-#include <libsolidity/interface/Exceptions.h>
 #include <libsolidity/inlineasm/AsmAnalysisInfo.h>
+#include <libsolidity/interface/ErrorReporter.h>
```

Comment:
I don't think so in this case.  It's not a reference to an ErrorReporter here.  It's a whole ErrorReporter object.